Assets/Scenes/Level2.unity

```diff
@@ -2543,14 +2876,14 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 5bc3a628348bea4408c521ac7ec602ab, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  noneMessage: 
-  magniferMessage: 
-  brushMessage: 
-  materialMessage: 
-  coveredMessage: 
-  magnifierClip: {fileID: 0}
-  materialClip: {fileID: 0}
-  tipManager: {fileID: 0}
+  noneMessage: NA
+  magniferMessage: Overall A simple design, but is not devoid of some decoration.
```

Comment:
"The design is simple and has few decorations."
or something like that

Reply:
Any particular reason for the rewording? Or just to shorten it?